Project.toml

```diff
@@ -16,9 +14,7 @@ OrderedCollections = "bac558e1-5e72-5ebc-8fee-abe8a469f55d"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 
 [compat]
-AWSCore = "0.5.2, 0.6"
-AWSSDK = "0.2, 0.3, 0.4, 0.5"
-AWSTools = "0.3.1, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1"
+AWS = "1"
 AutoHashEquals = "0.2.0"
 HTTP = "0.8.1"
```

Comment:
Hmmm... idk why this is here. We should remove this as we don't actually depend on `HTTP.jl`.
At this can be handled upstream.

```suggestion

```